repo2docker/app.py

```diff
@@ -347,7 +364,7 @@ def start(self):
             if self.build:
                 self.log.info('Using %s builder\n', bp.name,
                               extra=dict(phase='building'))
-                for l in picked_buildpack.build(self.output_image_spec):
+                for l in picked_buildpack.build(self.output_image_spec, self.build_memory_limit):
```

Comment:
should we make this a named argument? Just thinking we might want to add a CPU limit or other limits in the future and then I won't be able to remember in which order args go.

Reply:
Docker doesn't support CPU limits for builds yet, so IMO it's ok right now for this to be just a param that must be specified. We can always just screw ordering and say memory_limit=<whatever> when calling it regardless of it having a default value or not when we have too many of them I think?